website/assets/styles/pages/pricing.less

```diff
@@ -6,6 +6,10 @@
     max-width: 800px;
   }
 
+  @media (max-width: 767px) {
+    p {font-size: 16px;}
```

Comment:
FYI I've found it's usually best to use more specific selectors (easy to have selectors like `p` inadvertently leak)

For example, you might prefer `.comparison-paragraph` over `p`  (I'm not sure that's the right name in this case without looking at the HTML - just made it up as an example.  The other advantage to this is it makes your CSS more explicit since you can kinda tell what your rules are controlling more without having to look at the HTML file to get a sense for "what `p`s are there")